primo/opt_model/opt_model.py

```diff
@@ -155,6 +162,7 @@ def _add_variables(self):
                 campaign_set.append((campaign_id, n_well))
 
         model.v_q = pyo.Var(campaign_set, within=pyo.Binary)
+        model.B_slack = pyo.Var(within=pyo.NonNegativeReals)
```

Comment:
we were trying to follow the convention that all model variables have a name with a prefix v_. Could you please rename this to v_B_slack?